src/inc/quic_platform.h

```diff
@@ -66,17 +66,44 @@ typedef struct QUIC_SINGLE_LIST_ENTRY {
 //
 // Different pool tags used for marking allocations.
 //
+#ifdef _WIN32
+
+#define POOL_TAG_QUIC 'CIUQ'
+#define POOL_TAG_QCon 'noCQ'
+#define POOL_TAG_QCTP 'PTCQ'
+#define POOL_TAG_QStm 'mtSQ'
+#define POOL_TAG_QSBf 'fBSQ'
+#define POOL_TAG_QSFM 'MFSQ'
+#define POOL_TAG_QDta 'atDQ'
+#define POOL_TAG_QTst 'tsTQ'
+#define POOL_TAG_QPrf 'frPQ'
+#define POOL_TAG_QTol 'loTQ'
+
+#define QUIC_POOL_TAG(a, b, c, d) POOL_TAG_ ##a ## b ## c ## d
```

Comment:
`##a ## b ## c ## d` Is a having a different spacing from the b, c and d expected?

Reply:
There should be a space after the a for consistency, but functionally it doesn't matter.

Reply:
Actually Lars, the more I think about it, let's just ignore the warning and revert all this please.

Reply:
ACK